### DISCUSSION TITLE
Floating label for input=date

### DISCUSSION THREAD

**Discussion by pravinfullstack** [tailwindlabs/tailwindcss#5707] [Help] [answered]
I am stuck to create floating label for input date field. Any help would be highly appreciated.

Below is the stackoverflow question - 
https://stackoverflow.com/questions/69415819/floating-label-for-input-date

**Comment by simonswiss** [comment, marked as answer]
Hey!

I _think_ that `placeholder-shown` does nothing on the `input type="date"`, and your only options here is to:

1. flip the `type` attribute from `text` to `date` on focus/blur with JavaScript

or

2. possibly do some clever trickery with absolutely positioned elements of pseudo-elements, that mimic the placeholder.

I don't think either would be a bullet-proof solution, and not sure I'd recommend one over the other in particular. A quick google search shows you're definitely not the only person trying to do this, so you might find a few CodePen examples to take inspiration from 👍
